README.md

```diff
@@ -0,0 +1,100 @@
+# ec2
+
```

Comment:
Add .drone.yml and badge please :D

Reply:
Please use this `.drone.yml` file:

https://raw.githubusercontent.com/techservicesillinois/terraform-aws-acm-certificate/master/.drone.yml

And here is the code for the badge:

```
[![Build Status](https://drone.techservices.illinois.edu/api/badges/api/badges/status.svg)](https://drone.techservices.illinois.edu/api/badges)
```